src/frontend/src/lib/derived/tokens-to-display.derived.ts

```diff
@@ -0,0 +1,11 @@
+import { combinedDerivedSortedNetworkTokensUi } from '$lib/derived/network-tokens.derived';
+import { pointerEventStore } from '$lib/stores/events.store';
+import type { TokenUi } from '$lib/types/token';
+import { defineTokensToDisplay } from '$lib/utils/tokens-to-display.utils';
+import { derived, type Readable } from 'svelte/store';
+
+export const combinedDerivedTokensToDisplay: Readable<TokenUi[]> = derived(
+	[combinedDerivedSortedNetworkTokensUi, pointerEventStore],
```

Comment:
should we call this modules `tokens-ui.derived` and in another PR moved `combinedDerivedSortedNetworkTokensUi` here?

Reply:
yes, agreed!